packages/@react-stately/layout/src/TableLayout.ts

```diff
@@ -504,6 +518,53 @@ export class TableLayout<T> extends ListLayout<T> {
         }
         break;
       }
+      case 'section': {
```

Comment:
this case seems very similar to rowgroup, can we combine some logic?

Reply:
ya probably, this was a separated in a first pass to make sure the logic checked out